### PR TITLE
Awesome books: a complete website with navigation

### PR DESCRIPTION
**Pull request by NasKhalil**
This branch includes the following features implemented:

- Add a new file named `navigation.js`.
- Update `index.html` to add the navbar and the footer.
- Add three sections: 
-  - heading section
-  - add section
-  - contact section

### CODE REVIEW

**Review by o-rashad — CHANGES_REQUESTED** [review]
Asslam Alikom team, @NasKhalil  @zemola  

Hi 👋🏽, While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that doesn't fulfill one of the project `General Requirements` :
-  Make sure that you documented your work in a professional way. [LINK](https://github.com/microverseinc/curriculum-javascript/blob/main/books/m4_full_website.md#:~:text=make%20sure%20that%20you%20documented%20your%20work%20in%20a%20professional%20way.)
   > Please be sure to update your README with the project's information, so anybody who looks at your portfolio will be able to tell what the project is about. [Remember that this is the expected structure for a Microverse Project's README.](https://github.com/microverseinc/readme-template)

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Happy coding!👏:heart:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **Does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -22,7 +32,22 @@ <h2 class="project-title">Add a new book</h2>
...
+      <p>Do you have any questions or you just want to say "Hello"?</p>
+      <ul>
+        <li>Our e-mail: africadev@mail.com</li>
+        <li>Our phone number: 001122334455</li>
+        <li>Our address: The moon</li>
+      </ul>
+    </section>
+
+    <footer>
+      Copyright © AFRICADev 2021
```
Comment (SajjadAhmad14) [inline index.html:46]: I am really impressed with the work and effort you have put to make UI amazing. 
Please follow the wireframe for the footer. Footer text should be on the left side of the page.

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
## Required Changes ♻️
> "Change is inevitable. Change is constant. -Benjamin Disraeli"
## Hi Khalil and Azeem, 
 ### You have been doing a tremendous job so far :ok_hand:

## To Highlight 🎉
README is professional. :heavy_check_mark:
UI is facianating. :heavy_check_mark:
## To Improve 🙏

- [x] Please check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity!

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by SajjadAhmad14 — APPROVED** [review]
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:

### Hi, 
## Congratulations 🎉 Thank you for implementing changes suggested previously 🙏

## Happy Coding 👏👏👏
#### Please leave any questions or comments in the PR if you have any doubts or unclarity!
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**